src/App.js

```diff
@@ -1,5 +1,79 @@
-function App() {
-  return <div>Hello World!</div>;
+import { useState } from 'react';
+import styled from 'styled-components/macro';
+import GameForm from './components/GameForm';
+import Navigation from './components/Navigation';
+import GamePage from './pages/GamePage';
+import HistoryPage from './pages/HistoryPage';
+import { nanoid } from 'nanoid';
+
+export default function App() {
+  const [players, setPlayers] = useState([]);
+  const [nameOfGame, setNameOfGame] = useState('');
+  const [currentPage, setCurrentPage] = useState('play');
+  const [history, setHistory] = useState([]);
+
+  return (
+    <AppLayout>
+      <h1>Scorekeeper</h1>
+      {currentPage === 'play' && <GameForm onCreateGame={createGame} />}
```

Comment:
Maybe extract the pages to constants?

Reply:
How do you mean that exactly?

Reply:
`const PLAY="play"` outside of the component, or maybe even export it from another file as it is used in other places too

Reply:
Hmm I will think about it 💭 :)

Reply:
Good idea, Jan. That way we can prevent errors from typos.